assets/prefabs/items/CrudeShears.prefab

```diff
@@ -1,11 +1,16 @@
 {
   "parent": "engine:iconItem",
   "Item": {
-    "icon": "ManualLabor:ManualLabor#CrudeShears"
+    "icon": "ManualLabor:ManualLabor#CrudeShears",
+    "damageType": "shearDamage"
```

Comment:
FYI the `shovelDamage` you had here before comes from [CoreAssets](https://github.com/Terasology/CoreAssets/blob/develop/assets/prefabs/damageTypes/shovelDamage.prefab) and IIRC affects blocks based on the [soil template](https://github.com/Terasology/CoreAssets/blob/develop/assets/blocks/templates/soil.block).

So _if_ you want to specify a `damageType`, you'll need to create a respective new type in `CoreAssets` and associate it with a block template. [`leaf`](https://github.com/Terasology/CoreAssets/blob/develop/assets/blocks/templates/leaf.block) might be a suitable one.

Reply:
What is your opinion? do we want the shears to have damage?

Reply:
I think it would be nice to be able to cut down leaves faster and potentially get placeable leaf blocks out of it :thinking: